controllers/dynakube/dtclient_reconciler.go

```diff
@@ -32,9 +31,8 @@ type tokenConfig struct {
 }
 
 func (r *DynatraceClientReconciler) Reconcile(ctx context.Context, instance *dynatracev1beta1.DynaKube) (dtclient.Client, bool, error) {
-	now := r.Now
```

Comment:
wouldn't calling r.Now multiple times and different points in the function cause to have multiple different timestamps, couldn't that cause problems ?

Reply:
Do not see a problem there. - r.Now can't be called as it isn't a function. - And accessing the same field multiple times is not a problem.